### PR TITLE
bump doc version tags to 0.2.0

### PR DESCRIPTION
**Pull request by fgeorgatos**
this is just a w.i.p. PR

Also, link GitHub Renku repo header should point to https://datascience.ch/solutions/ instead? t.b.d.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -63,17 +63,17 @@
 
 # General information about the project.
 project = 'RENKU'
-copyright = '2017, Swiss Data Science Center'
-author = 'Eric Bouillet, Julien Eberle, Jiri Kuncar, Izabela Moise, Rok Roskar, Sofiane Sarni, Sandra Savchenko De Jong, Johann-Michael Thiebaut, Olivier Verscheure'
+copyright = '2018, Swiss Data Science Center'
```
Comment (rokroskar) [inline docs/conf.py]: 2017-2018?

Comment (fgeorgatos) [reply docs/conf.py]: yeap, that's the way

File: docs/conf.py [hunk 2]
```diff
@@ -63,17 +63,17 @@
 
 # General information about the project.
 project = 'RENKU'
-copyright = '2017, Swiss Data Science Center'
-author = 'Eric Bouillet, Julien Eberle, Jiri Kuncar, Izabela Moise, Rok Roskar, Sofiane Sarni, Sandra Savchenko De Jong, Johann-Michael Thiebaut, Olivier Verscheure'
+copyright = '2017-2018, Swiss Data Science Center'
+author = 'Eric Bouillet, Julien Eberle, Jiri Kuncar, Izabela Moise, Rok Roskar, Sofiane Sarni, Sandra Savchenko De Jong, Johann-Michael Thiebaut, Fotis Georgatos, Olivier Verscheure'
```
Comment (leafty) [inline docs/conf.py]: It was alphabetical

Comment (rokroskar) [reply docs/conf.py]: doh... I'll fix it in the docs PR I have open

**Pull request merged**